_test/shared/matlab_xunit/xunit/TestRunDisplay.m

```diff
@@ -169,14 +169,20 @@ function testRunFinished(self, did_pass)
 
             if did_pass
                 result = 'PASSED';
+                fh = 1;
```

Comment:
Change: 'fh ' should be expanded to a more meaningful name e.g. 'output_stream' or 'screen_outputID' I'd do something with a comment like:
`        screen_outputID = 1   % standard output to screen`
and
`        screen_outputID = 2   % standard error output to screen`